test/src/test/java/org/apache/accumulo/test/fate/zookeeper/ZooLockTest.java

```diff
@@ -274,42 +274,43 @@ public void testUnexpectedEvent() throws Exception {
     String parent = "/zltest-" + this.hashCode() + "-l" + pdCount.incrementAndGet();
 
     ConnectedWatcher watcher = new ConnectedWatcher();
-    ZooKeeper zk = new ZooKeeper(accumulo.getZooKeepers(), 30000, watcher);
-    zk.addAuthInfo("digest", "accumulo:secret".getBytes(UTF_8));
+    try (ZooKeeper zk = new ZooKeeper(accumulo.getZooKeepers(), 30000, watcher)) {
```

Comment:
Would be good to make more use of this being autoclosable but that could be a follow on ticket.

Reply:
I think these might be the only places we could use try-with-resources (I only fixed these because they generated warnings). But yes, separate issue, if there's anything else to be done with this.

Reply:
I looked again and it looks like this is the only use of this object other than ZooSession (where we are manually closing) so disregard.